pkg/dbmodels/dbmodels.go

```diff
@@ -167,6 +167,25 @@ const (
 	BastionSchemeTelnet BastionScheme = "telnet"
 )
 
+// Generic Helper
+func GenericNameOrID(db *gorm.DB, identifiers []string) *gorm.DB {
```

Comment:
@jwessel Can we make this function to return an error and handle it? this way we will be more Foolproof

Reply:
Actually *gorm.DB is not something that can fail, it's just an ORM constructor; the real SQL error could come after that, so it's better to keep this helper as light as possible